tests/test_operation_builder.py

```diff
@@ -40,12 +42,9 @@
 )
 from xdsl.traits import IsTerminator
 
-#  ____                 _ _
-# |  _ \ ___  ___ _   _| | |_
-# | |_) / _ \/ __| | | | | __|
-# |  _ <  __/\__ \ |_| | | |_
-# |_| \_\___||___/\__,_|_|\__|
-#
+################################################################################
```

Comment:
should we use `# region Results` with `# endregion` instead? lets us fold the code nicely

Reply:
Can we do that and keep the ASCII art stuff? I like ASCII art too much :grin:

Reply:
These ascii art should be removed sadly, they are a bit useless, and cannot be grepped :/
I'll think of `# region`, maybe we could do a pass in the code to change that everywhere.